guidelines/releases-guidelines/02-kyma-release-process.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Kyma release process
+label: internal
+---
+
+This document describes how to create a Kyma release. Start from defining release jobs as described in the [**Preparation**](#kyma-release-process-kyma-release-process-preparation) section. Then proceed to the [**Steps**](#kyma-release-process-kyma-release-process-steps).
+
+A Kyma release includes the following items:
+
+* Docker images for Kyma components
+* A GitHub release including release artifacts, such as source code and configuration
+* A Git tag
+* A release branch
+
+## Preparation
+
+> **NOTE:** This section applies only to new major and minor versions. If you release a patch, skip the preparation and go to the [**Steps**](#kyma-release-process-kyma-release-process-steps) section.
+
+To prepare a release:
+
+1. Create a release branch in the `kyma` repository. The name of this branch should follow the `release-x.y` pattern, such as `release-1.4`.
+
+   ```bash
+    git fetch upstream
+    git checkout -b release-{release_version} upstream/master
+    git push upstream release-{release_version}
+    ```
+
+   > **NOTE:** If you don't create the Kyma release branch at this point and add a  `post-rel{release_version}-kyma-release-candidate` post-submit job to the `test-infra` master, then pushing anything to the Kyma release branch, creating or rebasing the branch, triggers a new GitHub release.
+
+2. [Define new release jobs](#kyma-release-process-kyma-release-process-preparation-define-new-release-jobs) in the `test-infra` repository.
+3. [Remove old release jobs](#kyma-release-process-kyma-release-process-preparation-remove-old-release-jobs) in the `test-infra`.
+
+### Define new release jobs
+
+Define release jobs on the `master` branch in the `test-infra` repository. To ensure every job name is unique, prefix it with `pre-rel{versionNumber}`. Remember to provide the version number without any periods. For example, to find all jobs for the 0.9 release, look for job names with the `pre-rel09` prefix. To learn how to define a release job for a component, read the following [document](https://github.com/kyma-project/test-infra/blob/master/docs/prow/release-jobs.md).
```

Comment:
Should we change:
0.9 > `1.4`
pre-rel09 > `pre-rel14` ? 
since we were changing everything to `1.4` release version.